### ISSUE TITLE
need help to resolve flutter telr intégration

### ISSUE THREAD

**Issue by mohamedkhabir12** [BisoDevil/TelrFlutter#2]
hello, i contact you because i find many issus in the project if you can help me and i pay you please

**Comment by BisoDevil** [comment]
Ok sure, but please download Xcode 12.0 before we talk to save time

**Comment by mohamedkhabir12** [comment]
yes i already did , but if you have skype or facebook or anything to talk 
my skype id : live:khabir.mohamed12
facebook : https://www.facebook.com/mohamedkhabir1122/

**Comment by mohamedkhabir12** [comment]
my anydesk remote : 374 596 830

**Comment by BisoDevil** [comment]
I will call you in just an hour

**Comment by mohamedkhabir12** [comment]
ok great

**Comment by dinesh9936** [comment]
hello sir
will plz reply me

**Comment by safvancentrric** [comment]
hello sir , 
it cannot support null safety . I just clone and implemented null safety .
can I share code with you to update package with null safety implantation . or can i suggest any other way to telr payment implementation with null safety .

**Comment by BisoDevil** [comment]
sure make a pull request and update it

**Comment by safvancentrric** [comment]
remote: Permission to BisoDevil/TelrFlutter.git denied to safvancentrric.
fatal: unable to access 'https://github.com/BisoDevil/TelrFlutter.git/': The requested URL returned error: 403
PS D:\android\TelrFlutter>


also can't update with branch

**Comment by safvancentrric** [comment]
TelrFlutter.zip
<https://drive.google.com/file/d/11Y_ZShyZKQGCHGqU4FGYEdPEmsI9zX_M/view?usp=drive_web>
Sir its null safey implementation code  for telrFlutter. I cant create pull
request . so i send this code .
